scopes/component/ui/version-dropdown/version-dropdown.tsx

```diff
@@ -69,3 +84,127 @@ function VersionPlaceholder({ currentVersion, className }: { currentVersion?: st
     </div>
   );
 }
+type VersionMenuProps = {
+  tags: DropdownComponentVersion[];
+  snaps: DropdownComponentVersion[];
+  lanes: LaneModel[];
+  localVersion?: boolean;
+  currentVersion?: string;
+  latestVersion?: string;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+const VERSION_TAB_NAMES = ['TAG', 'SNAP', 'LANE'] as const;
+
+function VersionMenu({ tags, snaps, lanes, currentVersion, localVersion, latestVersion, ...rest }: VersionMenuProps) {
+  const [activeTabIndex, setActiveTab] = useState<number>(0);
+
+  const tabs = VERSION_TAB_NAMES.map((name) => {
+    switch (name) {
+      case 'SNAP':
+        return { name, payload: snaps };
+      case 'LANE':
+        return { name, payload: lanes };
+      default:
+        return { name, payload: tags };
+    }
+  }).filter((tab) => tab.payload.length > 0);
+
+  return (
+    <div {...rest}>
+      <div className={styles.top}>
+        <div className={styles.title}>
+          <span>Switch to tag or snap</span>
+        </div>
+        {localVersion && (
+          <NavLink
+            href={'?'}
+            className={classNames(
+              styles.versionLine,
+              styles.versionRow,
+              currentVersion === 'workspace' && styles.currentVersion
```

Comment:
extract this constant to the top of the file, as `const LOCAL_VERSION = 'workspace';`